### DISCUSSION TITLE
Tutorial videos

### DISCUSSION THREAD

**Discussion by Chris-Barber** [statiqdev/Discussions#47] [General]
Hi, are there are any tutorial videos anywhere to learn how to use Statiq?
Thanks

**Comment by daveaglick** [comment]
Not specifically (though I'd love to find time to make some). There have been a handful of videos recently though that include some intro-level content:

- https://www.youtube.com/watch?v=43oQTRZqK9g&feature=youtu.be
- https://www.youtube.com/watch?v=wHiTP-pS0r4
- https://www.youtube.com/watch?v=rCSppLf9dIM&feature=youtu.be

**Reply by Chris-Barber** [reply]
Thanks, I'll check them out
